fitbenchmarking/core/tests/test_fitting_benchmarking_starting_values.py

```diff
@@ -0,0 +1,174 @@
+"""
+Tests for fitbenchmarking.core.fitting_benchmarking.loop_over_starting_values
+"""
+from __future__ import (absolute_import, division, print_function)
+import inspect
+import os
+import unittest
+import mock
+
+from fitbenchmarking import mock_problems
+from fitbenchmarking.utils import fitbm_result, output_grabber
+from fitbenchmarking.core.fitting_benchmarking import loop_over_starting_values
+from fitbenchmarking.parsing.parser_factory import parse_problem_file
+from fitbenchmarking.utils.options import Options
+from fitbenchmarking.jacobian.SciPyFD_2point_jacobian import ScipyTwoPoint
+
+fitting_dir = "fitbenchmarking.core.fitting_benchmarking"
+
+
+# Due to structure of tests, some variables may not be previously defined
+# in the init function. Removes pytest dictionary iteration suggestion
+# pylint: disable=attribute-defined-outside-init, consider-iterating-dictionary
+def make_fitting_problem(file_name='cubic.dat', minimizers=None):
```

Comment:
Is this the same make_fitting_problem function which appears in a number of the other test python files? If yes should it be be factored out into its own .py?

Reply:
Yea it is used multiple places but this should be done in a separate issue, I will create one now for it